src/videosubscriberaccount.cs

```diff
@@ -0,0 +1,145 @@
+﻿//
+// VideoSubscriberAccount bindings
+//
+// Authors:
+//	Alex Soto  <alex.soto@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+using System;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.UIKit;
+
+namespace XamCore.VideoSubscriberAccount {
+
+	[Native]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[ErrorDomain ("VSErrorDomain")]
+	public enum VSErrorCode : nint {
+		AccessNotGranted,
+		UnsupportedProvider,
+		UserCancelled,
+		ServiceTemporarilyUnavailable,
+		ProviderRejected
+	}
+
+	[Native]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	public enum VSAccountAccessStatus : nuint {
+		NotDetermined,
+		Restricted,
+		Denied,
+		Granted
+	}
+
+	interface IVSAccountManagerDelegate { }
+
+	[Protocol, Model]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[BaseType (typeof (NSObject))]
+	interface VSAccountManagerDelegate {
+
+		[Abstract]
+		[Export ("accountManager:presentViewController:")]
+		void PresentViewController (VSAccountManager accountManager, UIViewController viewController);
+
+		[Abstract]
+		[Export ("accountManager:dismissViewController:")]
+		void DismissViewController (VSAccountManager accountManager, UIViewController viewController);
+	}
+
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[BaseType (typeof (NSObject))]
+	interface VSAccountManager {
+
+		[Field ("VSCheckAccessOptionPrompt")]
+		NSString CheckAccessOptionPromptKey { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		IVSAccountManagerDelegate Delegate { get; set; }
+
+		[Async]
+		[Export ("checkAccessStatusWithOptions:completionHandler:")]
+		void CheckAccessStatus (NSDictionary options, Action<VSAccountAccessStatus, NSError> completionHandler);
+
```

Comment:
Yes, it is already added check `VSAccountManager.cs` above or click here https://github.com/xamarin/xamarin-macios/pull/205/files/a2e2d7132e639d3878af25f6fe122a44d6cbc16d#diff-83977417e326fb6fdf44f1aee2b19528R17 :)

Added the Async and Sync versions of it